GETTING_STARTED_APP_DEV_CLI.md

```diff
@@ -0,0 +1,205 @@
+    AGENT_POOL='Hosted Ubuntu 1604' \
+    ARM_PROVIDER_STRICT=true \
+    BUILD_ARTIFACT_NAME='drop' \
+    BUILD_ARTIFACT_PATH_ALIAS='artifact' \
+    GO_VERSION='1.12.5' \
+    PIPELINE_ROOT_DIR='devops/providers/azure-devops/templates/infrastructure' \
+    REMOTE_STATE_CONTAINER='BACKENDSTATECONTAINERNAME' \
+    SCRIPTS_DIR='scripts' \
+    TEST_HARNESS_DIR='test-harness/' \
+    TF_DEPLOYMENT_TEMPLATE_ROOT='infra/templates/az-hello-world' \
+    TF_DEPLOYMENT_WORKSPACE_PREFIX='PROJECTDEPLOYMENTWORKSPACEPREFIX' \
+    TF_ROOT_DIR='infra' \
+    TF_VERSION='0.12.4' \
+    TF_WARN_OUTPUT_ERRORS=1
+
+# Build-specified Environment Variables. (Only creating a variable group
+# for the DevInt environment with this command execute multiple times for
+# every environment specified within the build pipeline definition). For
+# this getting started, we will only be creating a DevInt environment. We
+# also assume that we can reuse the Service Connection created as part of
+# the organization Cobalt template source Getting Started walkthrough.
+DEVINT_VAR_GROUP="DevInt $COBALT_VAR_GROUP_ENV_SUFFIX"
+az pipelines variable-group create --authorize true --name $DEVINT_VAR_GROUP --variables \
+    ARM_SUBSCRIPTION_ID='TARGETSUBSCRIPTIONID' \
+    REMOTE_STATE_ACCOUNT='BACKENDSTATESTORAGEACCOUNTNAME' \
+    SERVICE_CONNECTION_NAME='SERVICECONNECTIONNAME'
+
+# At this time, the Azure DevOps CLI does not support linking variable
+# groups to pipelines. We have a temporary workaround utilizing the Azure
+# DevOps invoke command to directly call the Azure DevOps REST API to
+# update the build definition.
+
+# Write the current value of the build pipeline definition to a temporary local
+# file, and save the PIPELINE_ID.
+az pipelines show --name $COBALT_PIPELINE_NAME -o json > builddef.json
+PIPELINE_ID=$(az pipelines show --name $COBALT_PIPELINE_NAME --query id)
+
+# Execute the list command to find the Variable Group IDs. Make note of
+# the IDs as they will need to be added to the build pipeline definition.
+az pipelines variable-group list
+
+# For the workaround, you'll be manually editing the builddef.json file
+# to add the variable group references. At the end of the file, you should
+# see the line "variableGroups" : null. Replace the null value with the
+# following, replacing the variable group ID placeholders with those found 
+# above:
+  "variableGroups": [
+      { "id": INFRAVARGROUP_ID },
+      { "id": DEVINTVARGROUP_ID }
+  ],
+
+# Save the file. Use the az devops invoke command to update the pipeline
+# build definition with the linked variable groups.
+az devops invoke --http-method PUT --area build --resource definitions --route-parameters project=$APP_DEVOPS_PROJECT_NAME definitionId=$PIPELINE_ID --in-file builddef.json
+
+#
+# 5. Run and verify 
+#
+
+# Queue a pipeline to run
+az pipelines run --name $COBALT_PIPELINE_NAME
+
+# At this point, the pipeline definition tears down any infrastructure
```

Comment:
This should probably better describe what changes would be expected to support the application.